build.sh

```diff
@@ -72,19 +72,25 @@ if [ ! -f "${BUILD_ENV}" ]; then
 fi
 source ${BUILD_ENV}
 
-# Clear out old builds
-if [ -d "${GEOPM_INSTALL}" ]; then
+# Check that GEOPM_INSTALL is defined unless we are skipping install
+if [ -z "${GEOPM_INSTALL}" ] && [ -z "${GEOPM_SKIP_INSTALL+x}" ];
+    echo "Please define GEOPM_INSTALL prior to using this script" 1>&2
+    exit 1
+fi
+
+# Clear out old builds unless we are skipping install
+if [ -d "${GEOPM_INSTALL}" ] && [ -z "${GEOPM_SKIP_INSTALL+x}" ]; then
     echo "Removing old build located at ${GEOPM_INSTALL}"
     rm -rf ${GEOPM_INSTALL}
 fi
 
-# e.g. To do a debug build, add --enable-debug to this env var
-GEOPM_GLOBAL_CONFIG_OPTIONS="${GEOPM_GLOBAL_CONFIG_OPTIONS} --prefix=${GEOPM_INSTALL}"
+# Append prefix to global config options unless skipping install
+if [ -z "${GEOPM_SKIP_INSTALL+x}" ]; then
+    GEOPM_GLOBAL_CONFIG_OPTIONS="${GEOPM_GLOBAL_CONFIG_OPTIONS} --prefix=${GEOPM_INSTALL}"
+fi
```

Comment:
Having the prefix set when we are not installing should not affect anything, right?  Why bother?

Reply:
It goes in the rpath of the compiled binaries.